### PR TITLE
add seq2seq benchmark

### PR DESCRIPTION
**Pull request by kmaehashi**

### CODE REVIEW

File: benchmarks/seq2seq/benchmark.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
+import numpy
```
Comment (asi1024) [inline benchmarks/seq2seq/benchmark.py]: Flake8 fails.
```F401 'numpy' imported but unused```

File: benchmarks/seq2seq/benchmark.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        updater = training.updaters.StandardUpdater(
+            train_iter, optimizer, converter=_convert, device=self.gpu)
+        trainer = training.Trainer(updater, (epoch, 'epoch'))
+
+        # Add extensions (for debugging)
+        trainer.extend(training.extensions.LogReport(
+            trigger=(1, 'epoch')))
+        trainer.extend(training.extensions.PrintReport(
+            ['epoch', 'iteration', 'main/loss', 'main/perp', 'elapsed_time']),
+            trigger=(1, 'epoch'))
```
Comment (asi1024) [inline benchmarks/seq2seq/benchmark.py]: these lines seem to should be executed in `__init__()`.

**Comment by kmaehashi** [comment]
Test

File: benchmarks/seq2seq/benchmark.py (hunk 1 above)
Comment (kmaehashi) [reply benchmarks/seq2seq/benchmark.py]: Fixed.

File: benchmarks/seq2seq/benchmark.py (hunk 2 above)
Comment (kmaehashi) [reply benchmarks/seq2seq/benchmark.py]: Ok, moved them to  ``__init__``.

**Comment by asi1024** [comment]
LGTM.

**Pull request merged**